packages/js/github-actions/actions/get-plugin-releases/action.yml

```diff
@@ -0,0 +1,24 @@
+name: Get Plugin L-2 release versions
+description: Get release versions for the L-2 releases from a specific Wordpress plugin like WooCommerce or Google Listing And Ads.
+
+inputs:
+  slug:
+    description: The plugin slug to get the versions.
+    required: true
+  releases:
+    description: Number of releases to get. By default L-2 directive (3 releases)
+    default: "3"
+  includeRC:
+    description: Weather to include Release Coadidates in the fetched versions. (Wordpress is Not compatible with this)
+    default: "false"
+  includePatches:
+    description: Weather to include Patches in the fetched versions. (Wordpress is Not compatible with this)
+    default: "false"    
+    
+outputs:
+  matrix:
+    description: A matrix array in the format ["7.4","7.5","7.6"]
+
+runs:
+  using: node16
```

Comment:
:grey_question: Out of curiosity, why 16?

Reply:
I was trying to be consistent with other project actions.

For example: 

https://github.com/woocommerce/grow/blob/trunk/packages/js/github-actions/actions/get-release-notes/action.yml
https://github.com/woocommerce/grow/blob/trunk/packages/js/github-actions/actions/update-version-tags/src/update-version-tags.js